src/components/custom-field-input-text/__snapshots__/custom-field-input-text.test.jsx.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Perhaps we need documentation in `contributing/testing.md` (another PR or make a new issue for it)

Reply:
Done @ https://github.com/mavenlink/design-system/issues/60